Fix warnings of " Unexpected console statement"

:brazil:: É usado o $console.err()$ no catch de algumas funções async, precisamos ou exibir uma mensagem de erro na tela infomando que o erro aconteceu ou configurar o eslint para não considerar isso um warning. Acredito que a primeira opção seria melhor. 

:us:: It is necessary to substitute $console.err()$ to an error message in the screen or config eslint to not consider this warning. I believe the first option would be nicer. 

```
 WARN  Compiled with 2 warnings                                                                                                                                              friendly-errors 17:35:32

Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):                                                                                                              friendly-errors 17:35:32

/opt/acompanha_legis/pages/deputados/index.vue
  202:9  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)

                                                                                                                                                                             friendly-errors 17:35:32
Module Warning (from ./node_modules/eslint-loader/dist/cjs.js):                                                                                                              friendly-errors 17:35:32

/opt/acompanha_legis/pages/proposicoes/_id.vue
  136:7  warning  Unexpected console statement  no-console

✖ 1 problem (0 errors, 1 warning)
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -19,5 +19,6 @@ module.exports = {
   ],
   // add your custom rules here
   rules: {
+    'no-console': 0,
   }
 }

--- a/components/app/TopMenu.vue
+++ b/components/app/TopMenu.vue
@@ -24,8 +24,8 @@
       <a-icon type="bell" />
       Notificaçōes
     </a-menu-item>
-    <a-dropdown class="fav-item-margin" v-if="$auth.loggedIn" key="favoritos">
-      <a-menu class="fav-list" slot="overlay" @click="handleMenuClick">
+    <a-dropdown v-if="$auth.loggedIn" key="favoritos" class="fav-item-margin">
+      <a-menu slot="overlay" class="fav-list" @click="handleMenuClick">
         <template v-if="$auth.user.favorite_proposicoes.length">
           <a-menu-item>
             <a-input
@@ -41,7 +41,7 @@
             </n-link>
           </a-menu-item>
         </template>
-        <a-menu-item class="no-favs" v-else>Não existem favoritos</a-menu-item>
+        <a-menu-item v-else class="no-favs">Não existem favoritos</a-menu-item>
       </a-menu>
       <a-button class="item-right">
         <a-icon type="heart" />Favoritos <a-icon type="down" />
@@ -56,11 +56,6 @@ export default {
       searchFav: ''
     };
   },
-  methods: {
-    handleMenuClick(e) {
-      console.log('click', e);
-    }
-  },
   computed: {
     favoriteProposicoes() {
       return (
@@ -71,6 +66,11 @@ export default {
           .toLowerCase()
           .includes(this.searchFav.toLowerCase());
       });
+    }
+  },
+  methods: {
+    handleMenuClick(e) {
+      console.log('click', e);
     }
   }
 };

